cogent/INSTALL.md

```diff
@@ -88,3 +90,15 @@ maintain a Stack build scheme.
   $> cabal test
 ```
 
+
+### Testing on macOS
+The default `cpp` bundled with macOS isn't compatible with Cogent – running `make examples` will fail.
+
+A solution:
+1. Install Homebrew
+2. Run `brew install gcc`. This will create symlinks `gcc-8` and `cpp-8` (or whatever the latest gcc version number is) in `/usr/local/bin` to the newly installed version of `gcc`.
+3. Provided `ls /usr/local/bin/cpp` outputs `No such file or directory`, it should be safe to run `ln -s /usr/local/bin/cpp-8 /usr/local/bin/cpp`.
+4. If `which cpp` doesn't print `/usr/local/bin/cpp` (ie. `/usr/local/bin` isn't early enough in the `PATH` environment variable) , then you can run the command `export PATH=/usr/local/bin/cpp-8:$PATH` in any shell where you want run the examples.
```

Comment:
I presume `/usr/local/bin/cpp-8` is the executable (ignore me if I'm wrong). I think people normally only put directories in the `PATH` variable.